template/roster-item.hbs

```diff
@@ -4,6 +4,7 @@
    <div class="jsxc-bar__caption jsxc-grow">
       <div title="{{jid}}" class="jsxc-bar__caption__primary">{{name}}</div>
       <div title="{{lastMessage}}" class="jsxc-bar__caption__secondary">{{lastMessage}}</div>
+      <div title="{{tag}}" class="jsxc-bar-tags"></div>
```

Comment:
To stay with the css BEM scheme, `jsxc-bar-tags` should be `jsxc-bar__tags`.

Reply:
changed